hacker_Rank/May13_Answer.py

```diff
@@ -0,0 +1,31 @@
+from algorithm import Algo
+
+obj = Algo()
+
+
+def solution(s, p, q):
+    print(s)
+    # A = 1
+    # C = 2
+    # G = 3
+    # T = 4
+    # K = 0
+    string = []
+    a =[]
+    m = len(p)
+
+    for i in range(m):
+        for j in range(p[i], q[i] + 1):
+            string.append(s[j])
+        minvalue = obj.min_value(string)
+        string = []
+        a.append(minvalue)
```

Comment:
Program is Wrong ,
use slicing technique,

Where is Impact numbers and its map to DNA